test/integration-test/commands/rename/rename-project.test.ts

```diff
@@ -0,0 +1,11 @@
+import { expect, test } from '@oclif/test'
+
+describe('User lists projects', () => {
+  test
+    .stdin('\n', 2000)
+    .stdout()
+    .command(['rename project', '-n', 'New Project Rename'])
```

Comment:
I though this command needed to have a project id as a first argument ?

Reply:
It could take the project id as an argument and if not provided it prompts the user with an inquirer prompt to select the desired project